utilities/test_suite/HOST/Tensor_misc_host.cpp

```diff
@@ -43,47 +42,51 @@ int main(int argc, char **argv)
     nDim = atoi(argv[4]);
     batchSize = atoi(argv[5]);
     numRuns = atoi(argv[6]);
-    BitDepthTestMode = atoi(argv[7]);
+    bitDepth = atoi(argv[7]);
     string dst = argv[9];
     string scriptPath = argv[10];
-    qaMode = (testType == UNIT_TEST); // unit test mode
+    qaMode = (testType == 0);
     bool axisMaskCase = (testCase == NORMALIZE || testCase == CONCAT);
     bool permOrderCase = (testCase == TRANSPOSE);
     int additionalParam = (axisMaskCase || permOrderCase) ? atoi(argv[8]) : 1;
     int axisMask = additionalParam, permOrder = additionalParam;
 
-    if (qaMode && batchSize != 3)
+    if(qaMode && batchSize != 3)
     {
-        cout<<"QA mode can only run with batchsize 3"<<std::endl;
+        cout<<"QA mode can only run with batchsize 3" << std::endl;
         return -1;
     }
 
     string funcName = augmentationMiscMap[testCase];
-    if (funcName.empty())
+    if(funcName.empty())
     {
         cout << "\ncase " << testCase << " is not supported\n";
         return -1;
     }
 
-    string func = funcName;
-    if (axisMaskCase)
-    {
-        char additionalParam_char[2];
-        std::snprintf(additionalParam_char, sizeof(additionalParam_char), "%d", axisMask);
-        func += "_" + std::to_string(nDim) + "d" + "_axisMask";
-        func += additionalParam_char;
-    }
-    if (permOrderCase)
+    std::string bitdepthStr; // Variable to store the bit depth as a string
+    switch (bitDepth)
     {
-        char additionalParam_char[2];
-        std::snprintf(additionalParam_char, sizeof(additionalParam_char), "%d", permOrder);
-        func += "_" + std::to_string(nDim) + "d" + "_permOrder";
-        func += additionalParam_char;
+        case 0: bitdepthStr = "u8"; break;
+        case 1: bitdepthStr = "f16"; break;
+        case 2: bitdepthStr = "f32"; break;
```

Comment:
1. We already have an enum for bit depths in C++ now. Pls discuss with Hazarath on how to use the same. I see i16_f32 might be extra. Can we augment that enum?

2. Any of this should ideally happen in the header. Once we come to the "Tensor_misc_host.cpp", we should just use the enum